docker/trustlines/README.md

```diff
@@ -38,7 +38,7 @@ docker-compose up --no-start
 
 We need the compiled contracts, which are installed in the relay server image. Copy them with:
 ```
-docker run --rm -it -v $(pwd):/tmp --entrypoint /bin/bash trustlines_relay -c "cp /usr/local/trustlines-contracts/build/contracts.json /tmp"
+docker-compose run --rm --no-deps -v $(pwd):/here --entrypoint /bin/bash relay -c "cp /opt/relay/trustlines-contracts/build/contracts.json /here"
```

Comment:
The directory being used in this docker-compose command does not matter for the next command.

Reply:
Okay yes I see. But I don't really see why you changed it.

Reply:
The docker command itself didn't work when I tried it, so I changed that to docker-compose.
Reusing /tmp may not be the best idea, because you may end up with temporary file in $(pwd) (well, not in this case when only calling cp)